Core/include/Acts/EventData/SourceLink.hpp

```diff
@@ -26,7 +25,7 @@ using geometry_id_t = decltype(std::declval<T>().geometryId());
 }  // namespace detail_sl
 
 class SourceLink final {
-  using any_type = std::any;
+  using any_type = AnyBase<16>;
```

Comment:
do you think it makes sense to typedef this to `ActsAny` or something like that? so we use a single byte width across the codebase

Reply:
That's already the case: https://github.com/acts-project/acts/blob/main/Core/include/Acts/Utilities/Any.hpp#L499

```cpp
using Any = AnyBase<sizeof(void*)>;
```